x/gov/handler.go

```diff
@@ -1,6 +1,7 @@
 package gov
 
 import (
+	fmt "fmt"
```

Comment:
:police_car:

Reply:
Somehow my goimports plugin changed when I updated to 1.11.5... Not so happy about this one either.